prov/efa/src/efa_hmem.c

```diff
@@ -156,6 +159,8 @@ static int efa_domain_hmem_info_init_cuda(struct efa_domain *efa_domain)
 				"Unable to register CUDA device buffer via dmabuf: %s. "
 				"Fall back to ibv_reg_mr\n", fi_strerror(-errno));
 			ibv_mr = ibv_reg_mr(g_device_list[0].ibv_pd, ptr, len, ibv_access);
+		} else {
+			info->dmabuf_supported = true;
```

Comment:
same thing, you need to check whether ibv_mr is NULL before claiming dmabuf support

Reply:
Also, try using  ofi_hmem_dmabuf_fd to get dmabuf fd

Reply:
Nah, ignore my first comment here, I think it's good for cuda (ibv_mr check)